simapp/simd/cmd/root.go

```diff
@@ -207,6 +200,40 @@ func addModuleInitFlags(startCmd *cobra.Command) {
 	crisis.AddModuleInitFlags(startCmd)
 }
 
+// genesisCommand builds genesis-related `simd genesis` command. Users may provide application specific commands as a parameter
+func genesisCommand(encodingConfig params.EncodingConfig, cmds ...*cobra.Command) *cobra.Command {
```

Comment:
This is not so useful if `genesisCoreCommand` is still in simapp.
We can move this one under `x/genutil/...` but do not import simapp (so, adding `runtime.BasicManager` as an argument and the `nodeHomePath`)

Reply:
that's a fair point, changes addressed